ecosystem/python/sdk/aptos_sdk/account_sequence_number.py

```diff
@@ -0,0 +1,178 @@
+# Copyright © Aptos Foundation
+# SPDX-License-Identifier: Apache-2.0
+import asyncio
+import logging
+import time
+from typing import Optional
+
+from aptos_sdk.account_address import AccountAddress
+from aptos_sdk.async_client import RestClient
+
+
+class AccountSequenceNumber:
+    """
+    A managed wrapper around sequence numbers that implements the trivial flow control used by the
+    Aptos faucet:
+    * Submit up to 50 transactions per account in parallel with a timeout of 20 seconds
```

Comment:
I think the number below now is 100 and not 50.  Similarly the sleep time is .01 and not .1

Reply:
right, right, I want to also tweak the delay... .01 is probably too aggressive and likely leads to more wasted networking.